src/components/Navbar/NavbarTabs/NavbarTabs.tsx

```diff
@@ -45,7 +45,7 @@ export const NavbarTabs = ({ navbarPageReload }: NavbarTabsProps) => {
   return (
     <Tabs
       data={navbarTabs}
-      value={!isDesktop || navbarPageReload ? -1 : activeTab}
```

Comment:
![image](https://github.com/lifinance/jumper.exchange/assets/43956540/ee5b69ae-f372-4974-9200-235f8404989d)

This change causes to show an active tab on the profile-page.

Reply:
Btw: In this ticket, I have touched the wrong value of -1 for the tabs. I found that `false` can be passed to deactivate the active tab. For that, the types needed some cleanup:

https://github.com/lifinance/jumper.exchange/pull/1029